test/CheckoutSdkTest/Marketplace/MarketplaceIntegrationTest.cs

```diff
@@ -111,5 +112,26 @@ private async Task ShouldUploadMarketplaceFile()
             fileResponse.ShouldNotBeNull();
             fileResponse.Id.ShouldNotBeNull();
         }
+
+        [Fact(Skip = "Unavailable in Sandbox")]
+        private async Task ShouldInitiateTransferOfFunds()
+        {
+            var createTransferRequest =
+                new CreateTransferRequest
+                {
+                    Source = new TransferSource {Amount = 100, Id = "ent_kidtcgc3ge5unf4a5i6enhnr5m"},
+                    Destination = new TransferDestination {Id = "ent_w4jelhppmfiufdnatam37wrfc4"},
+                    TransferType = TransferType.Commission
+                };
+
+            var createTransferResponse =
+                await FourApi.MarketplaceClient().InitiateTransferOfFunds(createTransferRequest);
+
+            createTransferResponse.ShouldNotBeNull();
+            createTransferResponse.Id.ShouldNotBeNullOrEmpty();
+            createTransferResponse.Status.ShouldNotBeNull();
+            createTransferResponse.Links.ShouldNotBeNull();
```

Comment:
Does this response contains Links? I didn't saw Links reference in docs

Reply:
Apparently yes, but it's hidden. https://github.com/checkout/checkout-api-reference/blob/master/nas_spec/components/schemas/Transfers/TransferLink.yaml